packages/terra-graphs-docs/src/terra-dev-site/graph/CoreConfiguration.b/Documentation/Axes.3.graph.mdx

```diff
@@ -80,7 +81,8 @@ axis: {
 | show                  | boolean        | `true`                     | no       | Toggle for showing the axis.                                                                                                                                                                                                                                          |
 | suppressTrailingZeros | boolean        | `false`                    | no       | Toggle to suppress tick values's trailing zeros when default d3 tick formatting is used. For X axis, this property works only when X axis type is set to AXIS_TYPE.DEFAULT. Specifying `~` in the tick format takes precedence over `suppressTrailingZeros` property. |
 | ticks                 | object         | `null`                     | no       | See [Ticks](./Ticks).                                                                                                                                                                                                                                                 |
-| type                  | string         | `AXIS_TYPE.DEFAULT`        | no       | See [type](#type). Normal number value or time-based. Only for x-axis.                                                                                                                                                                                                |
+| type                  | string         | `AXIS_TYPE.DEFAULT`        | no       | See [type](#type). Normal number value or time-based. Only for x-axis.                                                                                                                                                                                                |    
+| tickLabelsRotation    | integer        | -                          | no       | Sets the rotation of the tick labels to `0º` or `-45º`. Only for x-axis.                                                                                                                                                                                              |
```

Comment:
Shouldn't the default be 0?

Also, while the integer type implies that the corresponding integer should be used, formatting `0º` and `-45º` as code here is misleading because those aren't the values they'll be using.

Could say something like:
Sets the rotation of the x-axis tick labels to `0º` or `-45º`. Accepted values: `0` or `-45`